[#24 ] 관리자 계정에 대한 추가 및 삭제 기능 구현

donsonioc2010 opened the pull request:
## 기능 구현 내용
- 관리자 계정에 대한 추가
- 관리자 계정에 대한 삭제.

모두 기존에 제작한 Service를 재활용 하는방향으로 제작을 하였습니다.

donsonioc2010 committed:
feat : 관리자 계정 추가 및 삭제

donsonioc2010 committed:
test : 관리자 계정의 추가에 대한 테스트 작성

- 삭제의 경우에는 UserSignOutService를 그대로 사용하기에 추가적인 제작을 안함.

donsonioc2010 committed:
feat : 관리자 계정삭제기능 Controller 구현

donsonioc2010 committed:
Merge branch 'develop' into feature/#24-addAdminUser

donsonioc2010 committed:
Merge Rename으로 인한 수정

f-lab-kyle commented on src/main/java/com/nooblol/community/controller/AdminController.java:
```diff
@@ -0,0 +1,36 @@
+@RestController
+@RequestMapping("/admin")
+@RequiredArgsConstructor
+public class AdminController {
```
관리자와 유저의 차이가 뭘까요? 테이블이 분리되었다거나, 유저의 레벨(?)이 차이가 될까요?
일반 유저의 회원가입과 별 차이가 없어보여서요. 누구나 관리자로 회원가입을 할수 있는건가 싶기도 하고요 ㅎㅎ;

donsonioc2010 committed:
Merge branch 'develop' into feature/#24-addAdminUser

donsonioc2010 committed:
feat : Review적용 및 비효율적인 코드 리팩토링

- Session에 로그인정보를 담는 기능이 이후에 진행되어서, 로그인, 로그아웃기능을 개발한 시점에 해당 로직완성 하도록 진행 예정.
- Slf4j어노테이션 미사용중인 Service, Controller수정
- Exception은 표준 Exception을 사용하도록 수정
- Throw의 StackTrace로깅 확인이 불가능하던 곳들 수정
-
